src/views/profile-view/ProfileView.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="profile__container">
+    <img :src="profilebg" class="profile__bg" />
+    <j-box v-if="profile" p="800">
+      <j-flex a="start" direction="column" gap="500">
+        <div class="profile__avatar">
+          <j-avatar
+            style="--j-avatar-size: 100px"
+            :hash="$route.params.did"
+            :src="profile.profilePicture"
+          />
+          <j-button>Send message</j-button>
+        </div>
+        <j-text v-if="profile.familyName || profile.givenName" variant="heading-sm"> {{ `${profile.familyName} ${profile.givenName}` }}</j-text>
+        <j-text variant="heading-sm">@{{ profile.username }}</j-text>
+        <j-text variant="subheading"> {{ bio }}</j-text>
+      </j-flex>
+      <div class="grid"> 
+        <ProfileCard v-for="link in profileLinks" :key="link.id" :title="link.has_name" :description="link.has_description" />
+        <div class="add" @click="() => (showAddlinkModal = true)">
+          <j-icon name="plus" size="xl"></j-icon>
+          <j-text>Add Link</j-text>
+        </div>
+      </div>
+    </j-box>
+  </div>
+  <j-modal
+    size="lg"
+    :open="showAddlinkModal"
+    @toggle="(e) => toggleAddLinkModal(e)"
+  >
+    <ProfileAddLink></ProfileAddLink>
+  </j-modal>
+  <router-view></router-view>
+</template>
+
+<script lang="ts">
+import { ad4mClient } from "@/app";
+import { useDataStore } from "@/store/data";
+import { ExpressionTypes, Profile } from "@/store/types";
+import { getProfile } from "@/utils/profileHelpers";
+import { LinkExpression } from "@perspect3vism/ad4m";
+import { defineComponent } from "vue";
+import ProfileCard from './ProfileCards.vue'
+import ProfileAddLink from './ProfileAddLink.vue'
+import { useUserStore } from "@/store/user";
+
+export default defineComponent({
+  name: "ProfileView",
+  components: {
+    ProfileCard,
+    ProfileAddLink
+  },
+  data() {
+    return {
+      profile: null as null | Profile,
+      bio: "",
+      showAddlinkModal: false,
+      profileLinks: [] as any[],
+      profilebg: ""
+    }
+  },
+  methods: {
+    toggleAddLinkModal(e: any): void {
+      this.showAddlinkModal = e.target.open;
+    },
+  },
+  async mounted() {
+    const did = this.$route.params.did as string;
+    const me = await ad4mClient.agent.me();
+    const userStore = useUserStore();
+    const userPerspective = userStore.getFluxPerspectiveId;
+    
+
+    if (did === me.did) {
+      this.profile = userStore.getProfile!;
+    } else {
+      const profileLang = Object.values(useDataStore().neighbourhoods)[0]
+        .typedExpressionLanguages.find((t) => t.expressionType === ExpressionTypes.ProfileExpression)?.languageAddress;
```

Comment:
This is gonna fail if the user has not installed any neighbourhoods, I guess its hard to find an agent if you have not joined any neighbourhoods but even in this case there is no guarantee that the target user has joined that neighbourhood. 

Id instead use the PROFILE_EXPRESSION_OFFICIAL language and try to get their language from their. This can be seen as a public language as it is not templated specifically for a given neighbourhood.